cypress/integration/hostedWorkflows.js

```diff
@@ -0,0 +1,191 @@
+describe('Dockstore my workflows', function() {
+    require('./helper.js')
+
+    beforeEach(function() {
+        cy.visit(String(global.baseUrl) + "/my-workflows")
+    });
+
+    // Ensure tabs are correct for the hosted workflow, try adding a version
+    describe('Should be able to register a hosted workflow and add files to it', function() {
+      it('Register the workflow', function() {
+        // Select the hosted workflow
+        cy
+          .contains('dockstore.org')
+        cy.wait(500)
+        cy
+          .contains('dockstore.org')
```

Comment:
This was called twice because the .invoke('width').should('be.gt', 0) didn't return a usable object, but I couldn't get it to work and just forgot to take the first one out. I am changing it to use the invoke check so it will have to stay.